TRRandomizerCore/Editors/TR1ClassicEditor.cs

```diff
@@ -369,6 +368,10 @@ private void AmendTitleAndCredits(AbstractTRScriptEditor scriptEditor, TRSaveMon
             }
 
             bg.Save(editedTitle);
+
+            string titlePath = @"data\title.png";
```

Comment:
If this is `data\` should the `editedTitle` also be `data\title.png`? I am unfamiliar with how this part works, just stood out.

Reply:
`GetWriteBasePath` returns the WIP folder for randomizing (so something like `%localappdata%\trge\Edits\_ID_\Output`) so we just dump the image there and TRGE will shift it to where we specify in the gameflow, which is `titlePath`.

![image](https://github.com/LostArtefacts/TR-Rando/assets/33758420/3d9d36f4-cda0-43aa-8527-db858891eb58)

This part is a bit of a stopgap, we don't currently support webp - the title file was changed from PNG in V4. I'm not happy with the third party libraries I've found so far, so in the meantime, TRGE is just dumping the old PNG file in the backup folder to allow us to stamp it.